### PR TITLE
[SPTN-SECIMP-1630] Fix display image issue

### PR DESCRIPTION
**Pull request by DeepakIngale-WK**

### CODE REVIEW

File: MigraDoc/code/MigraDoc.DocumentObjectModel/MigraDoc.DocumentObjectModel.Shapes/Image.cs [hunk 1]
```diff
@@ -199,34 +199,36 @@ internal override void Serialize(Serializer serializer)
...
+	    if (!String.IsNullOrEmpty(workingDir))
+		    filePath = workingDir;
+	    else
+		    filePath = Directory.GetCurrentDirectory() + "\\";
+#else
+            throw new NotImplementedException();
+            //if (!String.IsNullOrEmpty(workingDir))
+            //    filePath = workingDir;
+            //else
+            //    filePath = Directory.GetCurrentDirectory() + "\\";
```
Comment (dchopra-lt) [inline MigraDoc/code/MigraDoc.DocumentObjectModel/MigraDoc.DocumentObjectModel.Shapes/Image.cs]: Remove Commented code in all the places which is not required, can keep the commented line which provide some message.

File: MigraDoc/code/MigraDoc.RtfRendering/MigraDoc.Improved.nuspec [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      <file src="..\..\..\lib\netstandard2.0\MigraDoc.Rendering.dll" target="lib\netstandard2.0\MigraDoc.Rendering.dll" />
+      <file src="..\..\..\lib\netstandard2.0\MigraDoc.Rendering.xml" target="lib\netstandard2.0\MigraDoc.Rendering.xml" />
+      <file src="..\..\..\lib\netstandard2.0\MigraDoc.RtfRendering.dll" target="lib\netstandard2.0\MigraDoc.RtfRendering.dll" />
+      <file src="..\..\..\lib\netstandard2.0\MigraDoc.RtfRendering.xml" target="lib\netstandard2.0\MigraDoc.RtfRendering.xml" />
+      <file src="..\..\..\lib\netstandard2.0\PdfSharp1_32.dll" target="lib\netstandard2.0\PdfSharp1_32.dll" />
+      <file src="..\..\..\lib\netstandard2.0\PdfSharp1_32.xml" target="lib\netstandard2.0\PdfSharp1_32.xml" />
+      <file src="..\..\..\lib\netstandard2.0\PdfSharp1_32.Charting.dll" target="lib\netstandard2.0\PdfSharp1_32.Charting.dll" />
+      <file src="..\..\..\lib\netstandard2.0\PdfSharp1_32.Charting.xml" target="lib\netstandard2.0\PdfSharp1_32.Charting.xml" />
+  </files>
+</package>
```
Comment (dchopra-lt) [inline MigraDoc/code/MigraDoc.RtfRendering/MigraDoc.Improved.nuspec]: Add line in the end.

File: MigraDoc/code/MigraDoc.RtfRendering/MigraDoc.RtfRendering.csproj [hunk 3]
```diff
@@ -2,6 +2,13 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
+    <OutputPath>..\..\..\lib</OutputPath>
+    <Version>1.33.0</Version>
+    <Version Condition="!($(SourceBranchName) == 'master' Or $(SourceBranchName) == 'TableRenderingPerformancePatch') And $(SourceBranchName) != ''">$(Version)-$(SourceBranchName.toLower()).$(BuildNumber)</Version>
```
Comment (dchopra-lt) [inline MigraDoc/code/MigraDoc.RtfRendering/MigraDoc.RtfRendering.csproj:7]: Check this condition is the branch should not be part of master as it compare with TableRenderingPerformancePatch (check with ST)

File: PDFsharp/code/PdfSharp/PdfSharp.Drawing/XImage.cs [hunk 4]
```diff
@@ -194,6 +194,21 @@ public static XImage FromFile(string path)
...
+    /// Silverlight supports PNG and JPEG only.
+    /// </summary>
+    /// <param name="stream">The stream containing a BMP, PNG, GIF, JPEG, TIFF, or PDF file.</param>
+    public static XImage FromStream(Stream stream)
+    {
+	    if (stream == null)
+		    throw new ArgumentNullException("stream");
+
+	    if (PdfReader.TestPdfFile(stream) > 0)
+		    return new XPdfForm(stream);
```
Comment (dchopra-lt) [inline PDFsharp/code/PdfSharp/PdfSharp.Drawing/XImage.cs:208]: The return type for the method is different, do XPdfForm return type support ?

File: azure-pipelines.yml [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+    relativePath: $(RelativePath)
+    buildConfiguration: $(BuildConfiguration)
+    buildRevisionNumber: $(BuildRevisionNumber)
+- template: /Build/task-templates/dotnet-cli/dotnet-cli-pack.yml@Twinfield
+  parameters:
+    relativePath: $(RelativePath)
+    buildRevisionNumber: $(BuildRevisionNumber)
+- template: /Build/task-templates/dotnet-cli/nuget-unregister-artifactory.yml@Twinfield
+  parameters:
+    feedName: $(nuget.feed.name)
```
Comment (dchopra-lt) [inline azure-pipelines.yml]: The pipeline should be part of this folder , please check the same with ST.

File: MigraDoc/code/MigraDoc.RtfRendering/MigraDoc.RtfRendering.csproj (hunk 3 above)
Comment (DeepakIngale-WK) [reply MigraDoc/code/MigraDoc.RtfRendering/MigraDoc.RtfRendering.csproj:7]: Showed this condition to System team.

File: PDFsharp/code/PdfSharp/PdfSharp.Drawing/XImage.cs (hunk 4 above)
Comment (DeepakIngale-WK) [reply PDFsharp/code/PdfSharp/PdfSharp.Drawing/XImage.cs:208]: XImage is parent class of XPdfForm

File: MigraDoc/code/MigraDoc.DocumentObjectModel/MigraDoc.DocumentObjectModel.Shapes/Image.cs (hunk 1 above)
Comment (DeepakIngale-WK) [reply MigraDoc/code/MigraDoc.DocumentObjectModel/MigraDoc.DocumentObjectModel.Shapes/Image.cs]: Removed

File: MigraDoc/code/MigraDoc.RtfRendering/MigraDoc.Improved.nuspec (hunk 2 above)
Comment (DeepakIngale-WK) [reply MigraDoc/code/MigraDoc.RtfRendering/MigraDoc.Improved.nuspec]: Added new line in the end

File: azure-pipelines.yml (hunk 5 above)
Comment (DeepakIngale-WK) [reply azure-pipelines.yml]: Added this file here jointly with the system team.

**Review by dchopra-lt — APPROVED** [review]

**Review by vgubar — APPROVED** [review]

**Pull request merged**